src-docs/src/views/flex/flex_example.js

```diff
@@ -328,6 +355,55 @@ export const FlexExample = {
         </div>
       ),
     },
+    {
+      title: 'Proportional shrinking of items',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: flexShrinkNumericSource,
+        },
+      ],
+      text: (
+        <p>
+          You can specify a number between 1 and 10 for each{' '}
+          <strong>OuiFlexItem</strong> to take up a proportional shrinking of
+          the <strong>OuiFlexGroup</strong> it is in. For this to apply: (1) the
```

Comment:
The grammar feels a little off to me. @kgcreative can u help please?

Reply:
I think item #2 is just when the shrinking is applied automatically but item #1 reads as if the user needs to set it manually. So I would change to:

You can specify a value 1--10 for each OuiFlexItem so it shrinks according to this value. Flex items shrink when the size of all items in the flex parent is larger than the parent's width. For flex shrinking to apply, you must set the `grow` property of the flex item to `false` or the `basis` property to `auto`.

Reply:
Sorry, @AMoo-Miki @kolchfa-aws. Should I rephrase those lines for better readability? 

@kolchfa-aws, in your comment above, are you suggesting I do something, or are you clarifying @AMoo-Miki what I am doing? Also, what are the links you are sending for?

Reply:
@BigSamu I am suggesting rewriting this paragraph as I specified above. The links were inserted by github, sorry, I just wanted to refer to parts that you specified as 1 and 2.